h_transport_materials/references.bib

```diff
@@ -2621,3 +2621,21 @@ @article{guthrie_permeation_1974
 volume = {53},
 year = {1974}
 }
+
+@article{shimada_tritium_2018,
+	title = {Tritium permeability measurement in hydrogen-tritium system},
+	volume = {129},
+	issn = {0920-3796},
+	url = {https://www.sciencedirect.com/science/article/pii/S0920379618301340},
+	doi = {10.1016/j.fusengdes.2018.02.033},
+	abstract = {Understanding of thermodynamic equilibria of multi-components hydrogen isotopes is required to accurately measure tritium permeability at the expected low tritium partial pressure and non-negligible high hydrogen partial pressure in a fusion blanket system. A gas-driven tritium permeation system that is capable of independently controlling hydrogen and tritium partial pressures was developed at Idaho National Laboratory to accurately measure low partial pressure tritium permeability. The thermodynamic equilibria for hydrogen (H) – tritium (T) permeation through metal are discussed to accurately measure tritium permeability, and the experimental conditions required for evaluating tritium permeability in H-T system are presented.},
+	language = {en},
+	urldate = {2022-04-28},
+	journal = {Fusion Engineering and Design},
+	author = {Shimada, M. and Pawelko, R. J.},
+	month = apr,
+	year = {2018},
+	keywords = {Hydrogen, Permeation, Tritium, Blanket, Reduced activation ferritic/martensitic steel},
+	pages = {134--139},
+	file = {ScienceDirect Full Text PDF:C\:\\Users\\remidm\\Zotero\\storage\\2GMYKNEK\\Shimada et Pawelko - 2018 - Tritium permeability measurement in hydrogen-triti.pdf:application/pdf;ScienceDirect Snapshot:C\:\\Users\\remidm\\Zotero\\storage\\ZK9KBPN2\\S0920379618301340.html:text/html},
+}
```

Comment:
I'm not sure if this is the right reference? this paper seems to refer to reduced activation steels rather than tungsten. Although we could add these too

Reply:
oh, let me check

Reply:
https://doi.org/10.1016/j.fusengdes.2019.03.083

This is the correct one...